internal/plugin/native/manager.go

```diff
@@ -585,6 +585,19 @@ func (rr *Manager) Source(name string) (api.Source, error) {
 	}
 }
 
+func (rr *Manager) GetSourcePlugin(name string) (plugin2.EXTENSION_TYPE, string, string) {
```

Comment:
GetPluginInfo seems similar to this one

Reply:
GetPluginInfo do not have the install infomation of the plugin
 GetSourcePlugin need the install infomation of the plugin